src/main/java/racingcar/racinggame/RacingGamePresenter.java

```diff
@@ -20,30 +23,44 @@ public void setView(RacingGameContract.View view) {
 
 	@Override
 	public void start() {
-		view.print(MessageType.INFO.message(GAME_START_MSG));
+		view.println(MessageType.INFO.message(GAME_START_MSG));
 
 		while (!racingGame.isEnd()) {
 			racingGame.play();
-			view.print(MessageType.INFO.message(racingGame.getPlayCount()+PLAY_START_MSG));
-			racingGame.getCars().forEach((car) -> view.printCarPositionBySymbol(car.getPosition())
+			view.println(MessageType.INFO.message(racingGame.getPlayCount()+PLAY_START_MSG));
+			racingGame.getCars().forEach((car) -> {
+						view.print(car.getName());
+						view.print(" :: ");
+						view.printCarPositionBySymbol(car.getPosition());
+					}
 			);
-			view.print(MessageType.INFO.message(racingGame.getPlayCount() + PLAY_END_MSG));
-			view.print(MessageType.EMPTY.message(""));
+			view.println(MessageType.INFO.message(racingGame.getPlayCount() + PLAY_END_MSG));
+			view.println(MessageType.EMPTY.message(""));
+		}
+		List<Car> winners = racingGame.getWinner();
+		String winnerCarMsg = "";
+		for( int i = 0; i < winners.size(); i++) {
+			winnerCarMsg = winnerCarMsg.concat(winners.get(i).getName());
+			if(i != winners.size() -1) {
+				winnerCarMsg = winnerCarMsg.concat(",");
+			}
 		}
```

Comment:
자바 컨벤션 위반한 부분이 있네요!
```suggestion
		for (int i = 0; i < winners.size(); i++) {
			winnerCarMsg = winnerCarMsg.concat(winners.get(i).getName());
			if (i != winners.size() - 1) {
				winnerCarMsg = winnerCarMsg.concat(",");
			}
		}
```
아래 링크들을 통해 컨벤션을 확인하실 수 있습니다 :)

https://google.github.io/styleguide/javaguide.html
https://myeonguni.tistory.com/1596